js-miniapp-sdk/src/types/responseTypes/AdResponse.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Response from native sdk for ads
+ * miniAppId is a GUID string
+ * adUnitId is string eg ca-app-pub-7941117952683310/1302674583
+ * startTime is ad's load start Time
+ * endTime is the ad's unload time
+ * provider is the active ad provider network selected
+ */
+export interface AdResponse {
+  miniAppId?: string;
+  adUnitId?: string;
+  startTime?: Date;
+  endTime?: Date;
+  adProvider?: string;
```

Comment:
I think that the Ad Provider shouldn't be exposed to the Mini App. The Host App will control selecting the Ad Provider, loading the Ad, and displaying the Ad so the Mini App shouldn't need to know what the add provider is. It shouldn't be given any more information than necessary.

Reply:
Removed!